smarts/core/mission_planner.py

```diff
@@ -202,44 +202,49 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
         return edge_ids
 
     def cut_in_waypoints(self, sim, pose: Pose, vehicle):
+
+        aggressivenes = 5
         radius = self._mission.task.trigger_radius
         neighborhood_vehicles = sim.neighborhood_vehicles_around_vehicle(
-            vehicle=vehicle, radius=radius
+            vehicle=vehicle, radius=80
         )
 
         position = pose.position[:2]
         lane = self._road_network.nearest_lane(position)
 
-        if neighborhood_vehicles:
-            nei_vehicle = neighborhood_vehicles[0]
+        if len(neighborhood_vehicles) == 0:
+            return []
 
-            target_position = nei_vehicle.pose.position[:2]
-            target_lane = self._road_network.nearest_lane(target_position)
+        nei_vehicle = neighborhood_vehicles[0]
+        target_position = nei_vehicle.pose.position[:2]
+        target_lane = self._road_network.nearest_lane(target_position)
 
-            offset = self._road_network.offset_into_lane(lane, position)
-            target_offset = self._road_network.offset_into_lane(
-                target_lane, target_position
-            )
+        offset = self._road_network.offset_into_lane(lane, position)
+        target_offset = self._road_network.offset_into_lane(
+            target_lane, target_position
+        )
 
-            if offset < target_offset + 5 and not self._task_is_triggered:
-                return []
+        cut_in_offset = 10 + aggressivenes
```

Comment:
May want to initialise speed limit variable

```suggestion
        cut_in_offset = 10 + aggressiveness
        speed_limit = 0
```

Reply:
Thanks,